### PR TITLE
Add more expression on doc.go for bitutil package

### PR DESCRIPTION
**Pull request by andybclee**
## Proposed changes

- Write more description for common/bitutil package
- This is a documentation PR

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: common/bitutil/doc.go [hunk 1]
```diff
@@ -10,5 +10,41 @@
 //
 // TODO-Klaytn: put the original LICENSE file in the common/bitutil directory
 
-// Package bitutil implements fast bitwise operations.
+/*
+Package bitutil implements fast bitwise operations and compression/decompressions
```
Comment (ehnuje) [inline common/bitutil/doc.go]:
```suggestion
Package bitutil implements fast bitwise operations and compression/decompressions.
```

File: common/bitutil/doc.go [hunk 2]
```diff
@@ -10,5 +10,41 @@
...
+    `Safe` means it can be performed on all architectures
+    `Fast` means it only can be performed on architecture which supports unaligned read/write
+
+Compression and Decompression
+
+Following operations are supported
+  - CompressBytes
+  - DecompressBytes
+
+How compression works
```
Comment (ehnuje) [inline common/bitutil/doc.go]: There is no `How decompression works`, can we have one?

File: common/bitutil/doc.go [hunk 3]
```diff
@@ -10,5 +10,41 @@
...
+
+The compression algorithm implemented by CompressBytes and DecompressBytes is
+optimized for sparse input data which contains a lot of zero bytes. Decompression
+requires knowledge of the decompressed data length.
+
+	Compression works as follows:
+
+	  if data only contains zeroes,
+		  CompressBytes(data) == nil
+	  otherwise if len(data) <= 1,
```
Comment (laggu) [inline common/bitutil/doc.go]: what about `else if` instead of `otherwise if`

Comment (andybclee) [reply common/bitutil/doc.go]: @laggu Actually it was copied from the code comments. =)

File: common/bitutil/doc.go (hunk 2 above)
Comment (andybclee) [reply common/bitutil/doc.go]: Basically decompression works in reverse way of compression. Anyway I'll try to add more description for decompression.

**Pull request merged**